packages/version.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-SCRIPTPATH="$(cd "$(dirname "$0")"; pwd -P)"
-RELEASE="$(cd ${SCRIPTPATH}; cat ../configure.ac|grep C_INIT|cut -c 20-|rev|cut -c 3-|rev)"
-MAJOR_RELEASE="$(cd ${SCRIPTPATH}; cat ../configure.ac|grep C_INIT|cut -c 20-|rev|cut -c 3-|rev|cut -d. -f1)"
-REVISION="$(cd ${SCRIPTPATH}; git rev-list --all |wc -l | tr -d '[[:space:]]')"
+SCRIPTPATH="$(cd "$(dirname "$0")" || exit 1; pwd -P)"
```

Comment:
Why `exit` ?

Reply:
Actually, I think that in `SCRIPTPATH` there is no need to exit.  
For other variables, it makes sense because if we fail to change the directory, there is no need to evaluate the variables.  
However, since there are no destructive commands, this might not be necessary either.

Reply:
It complicates the code quite a bit. :cat:

Reply:
Maybe we should keep `exit` for cases when some `cd` is unseccessful?
It won't cause any issues in normal cases, but might prevent errors in defining the variables.

Reply:
I didn't see your last response when I was writing mine.

Reply:
I like the idea in general. But as it's done now it complicates the code a bit. Anyway, let's do it as you've proposed. :)